ci-operator/step-registry/ipi/deprovision/gcp/aws-route53/ipi-deprovision-gcp-aws-route53-ref.metadata.json

```
@@ -0,0 +1,11 @@
+{
+	"path": "ipi/deprovision/gcp/aws-route53/ipi-deprovision-gcp-aws-route53-ref.yaml",
```

```suggestion
	"path": "ipi/deprovision/aws/route53/ipi-deprovision-aws-route53-ref.yaml",
```
This day2 operation can be re-used by other platforms, so we can remove `gcp`.

---

How about moving to `ci-operator/step-registry/aws/deprovision/route53/public-record-sets/`? 

FYI what `ci-operator/step-registry/aws/deprovision/route53/` has now:
```
$ tree ci-operator/step-registry/aws/deprovision/route53/
ci-operator/step-registry/aws/deprovision/route53/
├── OWNERS
└── private-hosted-zone
    ├── aws-deprovision-route53-private-hosted-zone-commands.sh
    ├── aws-deprovision-route53-private-hosted-zone-ref.metadata.json
    ├── aws-deprovision-route53-private-hosted-zone-ref.yaml
    └── OWNERS

2 directories, 5 files
$

---

Sure, sounds good to me.